site.yml

```diff
@@ -203,6 +203,30 @@
       when: ceph.enabled
       tags: ['ceph','ceph-client']
 
+- name: v7k config
```

Comment:
Could we put the tasks in storwize role in v7k_integration.yml in [cinder-data role](https://github.com/blueboxgroup/ursula/tree/master/roles/cinder-data/tasks), instead of running it as a individual role? To keep consistent.